test/distributed/checkpoint/test_checkpoint.py

```diff
@@ -198,6 +198,10 @@ def write_data(
     def finish(self, metadata: Metadata, results: List[List[WriteResult]]) -> None:
         self._fail_rank("fail_finish")
 
+    @staticmethod
+    def check(checkpoint_id: Union[str, os.PathLike]) -> bool:
```

Comment:
nit: `check` could be considered a little vague, what about something like `is_valid_checkpoint_id` or `is_valid`?